ingeniamotion/wizard_tests/brake.py

```diff
@@ -12,7 +12,7 @@
     from ingeniamotion import MotionController
 
 
-class Brake(BaseTest[None]):
+class Brake(BaseTest[None]):  # type: ignore [type-var]
```

Comment:
No results for this test?

Reply:
The Brake test is a special one, it should be started and ended, instead of the other test, they should only be run.